jobs/JJOB_DA_3DENVAR

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash -l
+set -e
+
+cat <<EOF
+#================================================================================
+#================================================================================
+# JJOB_DA_3DENVAR
+#   Run SOCA 3DVAR, checkpoint, forecast
+#================================================================================
+#================================================================================
+EOF
+module purge
+module use -a /scratch2/NCEPDEV/marine/marineda/modulefiles/
+module load jedi-intel-17.0.5.239
+
+cd $RUNCDATE
+
+cat <<EOF
+#============================================
+# Link ensemble members for ENVAR
+#============================================
+EOF
+# link files here
+
+cat <<EOF
+#============================================
+# Run SOCA 3DENVAR
+#============================================
+EOF
+export OOPS_TRACE=0
+srun -n $NPE ${SOCA_EXEC}/soca_3dvar.x ./yaml/3dhybenvar.yml
+
+cat <<EOF
+#============================================
+# Checkpoint SOCA 3DENVAR analysis
+#============================================
+EOF
+if   [ ! -d RESTART ]; then mkdir -p RESTART; fi
+srun -n $NPE ${SOCA_EXEC}/soca_checkpoint_model.x ./yaml/checkpointmodel.yml
+
+cat <<EOF
+#========================================================
+# Prepare restarting deterministic forecast from analysis
+#========================================================
+EOF
+if [ -d ANA ]; then rm -Rf ANA; fi
+if [ ! -d ANA ]; then mkdir -p ANA; fi
+ln -s $PWD/INPUT_MOM6/* ./ANA/
+# Unlink old restarts
+rm -f ./ANA/MOM.*.nc
+rm -f $PWD/ANA/cice_bkg.nc
+# Get MOM's restart
+mv -f RESTART/MOM.*.nc ./ANA/  # Because checkpoint dumps analysis in RESTART
+ln -s $PWD/Data/cic.socagodas.an.*.nc ./ANA/cice_bkg.nc
+
+cat <<EOF
+#============================================
+# Run mom6 only deterministic forecast
```

Comment:
Shouldn't this be at the forecast job?

Reply:
Yes, it should. But that need to be addressed in a different pr, the 3dvar is already implemented this way.